### DISCUSSION TITLE
Flatline rule Aggregation

### DISCUSSION THREAD

**Discussion by fast-turtle-9** [jertel/elastalert2#1390] [Q&A] [answered]
I am trying to get a flatline rule to aggregate and bring all matches in the last 30 minutes, and my rule checks to see if there is flatline every 15 minutes, is that supposed to work?
I can share the yaml file as well if need be.

**Comment by jertel** [comment, marked as answer]
Aggregation is useful for grouping results together and perhaps performing a calculation on that grouping, such as an average, max, min, etc. Since a flatline is only triggering on the absence of data there isn't anything to aggregate.
